test/acceptance/features/bindAppToService.feature

```diff
@@ -415,113 +415,6 @@ Feature: Bind an application to a service
         Then Service Binding "service-binding-c-e" is ready
         And The application env var "HOST_ADDR" has value "127.0.0.1:8080"
 
-    @olm
-    Scenario: Creating binding secret from the definitions managed in OLM operator descriptors
```

Comment:
Why is this scenario being removed? Isn't the case of filling the application in the ServiceBinding below?

It seems this scenario didn't provide an application because its presence didn't influence on the results being observed, so removing this might not be what we want, unless it has been completely deprecated.